### ISSUE TITLE
Find examples for elements of the standard

### ISSUE THREAD

**Issue by ericherman** [publiccodenet/community-implementation-guide-standard#102]
What should a README contain?
What should a CONTRIBUTING file contain?
What should a GOVERNANCE file contain?
What should a ChangeLog file contain?

Find two examples for each file showing "minimum to meet the bar" and "best of class" real-world examples.

**Comment by Ainali** [comment]
Would we want "minimum to meet the bar" to be a file that only meets MUST requirements (and has good reasons to not meet the SHOULDs) or would it be a file that meets both MUST and SHOULD requirements and doesn't have the need for explanations/exceptions?

**Comment by ericherman** [comment]
To me, a codebase having good circumstantial reasons for SHOULD exceptions is a different axis than understanding completeness between minimum-to-meet and best-of-class.

Thus, I think ideally it would be all MUST and all SHOULD, to avoid confusing those two dimensions.

However, if there happen to be SHOULD requirements which are explained as exceptions, I think that could be fine.

But, let's see what we find.

**Comment by Ainali** [comment]
I agree. And then logically, a "best of class" then meets all OPTIONAL as well, right?

**Comment by ericherman** [comment]
Given that most OPTIONAL are things that we would recommend, then I'd say likely, yes.
